doc/Requirements Specification/Bibliography.bib.bak

```diff
@@ -0,0 +1,11 @@
+% This file was created with JabRef 2.11.1.
+% Encoding: Cp1252
+
+
+@Electronic{tuChemPcm,
+  Title                    = {Palladio Component Model},
+  Month                    = {09},
+  Url                      = {https://www.tu-chemnitz.de/informatik/ST/research/projects/palladio-component-model},
+  Year                     = {2015}
+}
+
```

Comment:
You included the `.bak` file. Please exclude it.

Consider excluding `**/*.bak` in the global `.gitignore`